boards/xtensa/m5stack_core2/m5stack_core2.dts

```diff
@@ -81,7 +86,7 @@
 
 &i2c0 {
 	status = "okay";
-	clock-frequency = <I2C_BITRATE_STANDARD>;
+	clock-frequency = <I2C_BITRATE_FAST>;
```

Comment:
Is the bitrate change really needed here?

Reply:
It was something I noticed during this change. All connected devices support 400kbit and I want to make sure high responsiveness. I created a separate commit for clarification.